docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,163 @@
+# Reusing a policy trained with Garage
+
+In this section you will learn how to save and load snapshots of your experiment
+while training. We will also show you how to extract a trained policy from a
+snapshot.
+
+## Saving an experiment snapshot
+
+As you've seen in previous guides, experiment activities are governed by a
```

Comment:
if you refer to other docs pages, you should always link to them.

Reply:
I don't think the one I need to link to is on master yet, so I'll come back to this PR later in the week.